rosette/src/RBLstream.cc

```diff
@@ -364,6 +366,28 @@ DEF("prim-print",obPrint, 1, MaxArgs)
 }
 
 
+DEF("ostream-log-time",obLogTime, 1, 1)
+{
+    CHECK(0, Ostream, strm);
+    if (strm->stream) {
+        char buf[128];
+
+        time_t rawtime;
+        struct tm * timeinfo;
+        time (&rawtime);
+        timeinfo = localtime(&rawtime);
+        strftime(buf, sizeof(buf),"%Y-%m-%d %I:%M:%S\n",timeinfo);
+
+        if (errno = fprintf(strm->stream, "%s", buf))
+            return FIXNUM(-errno);
```

Comment:
Braces.  I know Greg may not have used them elsewhere, but leaving out braces so often leads to bugs that we're enforcing it.